config/software/cacerts.rb

```diff
@@ -29,13 +29,18 @@
 build do
   ship_license "https://www.mozilla.org/media/MPL/2.0/index.815ca599c9df.txt"
 
-  mkdir "#{install_dir}/embedded/ssl/certs"
+  if windows?
+    mkdir "#{python_2_embedded}/ssl/certs"
+    copy "#{project_dir}/cacert.pem", "#{python_2_embedded}/ssl/certs/cacert.pem"
+    copy "#{project_dir}/cacert.pem", "#{python_2_embedded}/ssl/cert.pem"
+
+    mkdir "#{python_3_embedded}/ssl/certs"
+    copy "#{project_dir}/cacert.pem", "#{python_3_embedded}/ssl/certs/cacert.pem"
+    copy "#{project_dir}/cacert.pem", "#{python_3_embedded}/ssl/cert.pem"
```

Comment:
Should we guard these with `if with_python_runtime? "2"` (resp. 3)? That way we won't create the `embedded2` dir in an Agent 7 build, for instance.

Reply:
Made the change.